libraw/src/main/java/com/homesoft/photo/libraw/LibRaw.java

```diff
@@ -120,6 +151,37 @@ public Bitmap decodeBitmap(BitmapFactory.Options options) throws ErrnoException
         return b;
     }
 
+    /**
+     * Get a list of out-of-camera white balance multipliers. To set these, call setUserMul
+     * with the r, g, b, g1 values. Results may have color temp in kelvin attached or not,
+     * depending on what data is included in the raw file.
+     *
+     * @return a list of {@link CameraWhiteBalance} objects representing the camera's preset white
+     * balance values
+     */
+    public List<CameraWhiteBalance> getCameraWhiteBalanceMultipliers() {
```

Comment:
ah, so separating them out into WB_coeffs and WBCT_coeffs? I'm down with that. generally from that page it seemed like the latter is just the former tagged w/ color temps, so that's why I combined them, but given they come from different sources, absolutely open to separating them out. Will update.